lean4-infoview/src/infoview/info.tsx

```diff
@@ -186,7 +186,7 @@ export function InfoDisplay(props0: InfoDisplayProps) {
                         Expected type {sortButton} {filterButton}
                     </summary>
                     <div className='ml1'>
-                        {hasTermGoal && <GoalUi pos={pos} goal={termGoal} filter={goalFilters} />}
+                        {hasTermGoal && <GoalUi pos={pos} goal={termGoal} filter={goalFilters} index={0} />}
```

Comment:
@EdAyers not sure if this is the right fix. Maybe it should be an optional prop?